fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ParameterVisitorBatchDAO.java

```diff
@@ -399,10 +399,11 @@ public void visit(QuantityParmVal param) throws FHIRPersistenceException {
         BigDecimal quantityLow = param.getValueNumberLow();
         BigDecimal quantityHigh = param.getValueNumberHigh();
 
-        // XXX why no check for isBase on this one?
+        // XX why no check for isBase on this one?
```

Comment:
I sort of get the `XXX` to `XX` change, but `XXX` is a built-in marker for eclipse (sort of like a TODO).
Anyway, what would be nice is if we could just replace the question with the answer.

My guess is that we skip the isBase check just because we happen to know there are no cross-resource/global search parameters of type quantity.
@punktilious does that sound right to you?

Reply:
Options:
A. just replace the existing question like "No isBase check because there are no system-level quantity search parameters"
B. add a check for isBase and throw an UnsupportedOperationException (or similar)
C. add proper support for system-level quantity parameters

I think maybe B is the best for now because I don't think its important to add, but I think it might be better to error out than to store it as a resource-type-specific param...at least when we actually start using the global search param tables (via #1921)

Reply:
I think B too...as long as we're absolutely confident we can't inadvertently hit this. It seems a little over-the-top to make it a configurable (fhir-server-config flag) error vs. warning.

I'm just starting to look at properly supporting system-level parameters.

Reply:
There's a couple other parameter types that skip the isBase check, so I added a comment "No isBase check because there are no system-level xxxxxxx search parameters".  This was option A.  If option B is preferred, I can throw an exception instead.

Reply:
Updated to option B.